ACMILabs/playlist-label
# #81 Update npm packages

Resolves #81

Update npm packages.

### Acceptance Criteria
- [x] `npm update`

### Relevant design files
* None

### Testing instructions
1. See this build still passes
1. Run `npm audit` to see that there are no vulnerabilities

### DoD
For requester to complete:
~- [ ] New logic has been documented~
~- [ ] New logic has appropriate unit tests~
~- [ ] Deployment / migration instruction have been updated if required~
